scripts/create_single_recipient_setup.py

```diff
@@ -11,16 +11,20 @@
 from brownie import AllowedRecipientsBuilder
 
```

Comment:
please add comment on how to use tokens and tokens registry params
and for grant_rights param